### PR TITLE
Tweet after publish

### LINKED ISSUE #865: WEB - Tweet CTA after Publish + Gate

### PR DESCRIPTION
**Pull request by ikjudd**
- Closes #865

### CODE REVIEW

File: js/sdk/src/components/ReleaseCreate.js [hunk 1]
```diff
@@ -545,10 +583,27 @@ const ReleaseCreate = ({ canAddContent, hubPubkey }) => {
             </NpcMessage>
           </Box>
         )}
+      {releaseCreated && (
+        <ReleaseCreateSuccess
+          releasePubkey={releasePubkey}
+          hubReleaseKey={releaseInfo?.hubRelease?.toBase58()}
```
Comment (ejf89) [inline js/sdk/src/components/ReleaseCreate.js:589]: **`hubReleasePubkey`

File: js/sdk/src/components/ReleaseCreateSuccess.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+          variant="h4"
+          align="left"
+          marginLeft={'auto'}
+          marginRight={'auto'}
+          marginBottom={'16px'}
+          width={'680px'}
+        >
+          {`${artist} -`} <i>{`${title}`}</i> {`has been created.`}
+        </Typography>
+        <ReleaseSuccessBox columns={'repeat(2, 1fr)'}>
```
Comment (ejf89) [inline js/sdk/src/components/ReleaseCreateSuccess.js:44]: columns should be declared in styled object below

File: js/sdk/src/components/ReleaseCreateSuccess.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  )
+}
+const ReleaseSuccessContainer = styled(Box)(() => ({
+  height: '100%',
+}))
+
+const ReleaseSuccessBox = styled(Box)(() => ({
+  display: 'grid',
+  width: '765px',
+  // minHeight: '547px',
```
Comment (ejf89) [inline js/sdk/src/components/ReleaseCreateSuccess.js:122]: remove commented code plz

File: js/sdk/src/components/ReleaseSettingsModal.js [hunk 4]
```diff
@@ -66,7 +66,11 @@ const ReleaseSettingsModal = ({
             {userIsRecipient && (
               <Royalty releasePubkey={releasePubkey} release={release} />
             )}
-
+            <ShareToTwitter
+              artist={metadata.properties.artist}
+              title={metadata.properties.title}
+              url={metadata.external_url?.slice(8)}
```
Comment (ejf89) [inline js/sdk/src/components/ReleaseSettingsModal.js:72]: why are we using the `.slice(8)` on the url?
if its required for the link to behave properly, it should be done in the component for reusability

**Review by ejf89 — CHANGES_REQUESTED** [review]
plz address comments

File: js/sdk/src/components/ReleaseSettingsModal.js (hunk 4 above)
Comment (facing-n) [reply js/sdk/src/components/ReleaseSettingsModal.js:72]: yeah this is confusing - if you are trying to remove `https://` you should probably use replace so it is more explicit as to what the purpose is.... i also just dont think you need to remove that at all and that is fine in the tweet. all tweets turn into tiny links anyway so all links end up taking up the same amount of characters

**Review by ejf89 — APPROVED** [review]

**Pull request merged**